tests/codes/process/test_solver.py

```diff
@@ -96,70 +99,134 @@ def test_get_species_fraction_retrieves_parameter_value(self):
 class TestSolverIntegration:
     DATA_DIR = Path(Path(__file__).parent, "test_data")
+    MODULE_REF = "bluemira.codes.process._setup"
 
     def setup_method(self):
         self.params = ProcessSolverParams.from_json(utils.PARAM_FILE)
 
+        self._indat_patch = mock.patch(f"{self.MODULE_REF}.InDat")
+
+    def teardown_method(self):
+        self._indat_patch.stop()
+
     @pytest.mark.longrun
-    def test_run_mode_outputs_process_files(self):
-        run_dir = tempfile.TemporaryDirectory()
-        build_config = {"run_dir": run_dir.name}
-        solver = Solver(self.params, build_config)
+    def test_run_mode_outputs_process_files(self, tmp_path):
+        solver = Solver(self.params, {"run_dir": tmp_path})
 
-        solver.execute(RunMode.RUN)
+        with contextlib.suppress(CodesError):
+            solver.execute(RunMode.RUNINPUT)
 
-        assert Path(run_dir.name, "IN.DAT").exists()
-        assert Path(run_dir.name, "MFILE.DAT").exists()
+        assert Path(tmp_path, "IN.DAT").exists()
+        assert Path(tmp_path, "MFILE.DAT").exists()
 
     @pytest.mark.parametrize("run_mode", [RunMode.READ, RunMode.READALL])
     def test_read_mode_updates_params_from_mfile(self, run_mode):
         # Assert here to check the parameter is actually changing
         assert self.params.r_tf_in_centre.value != pytest.approx(2.6354)
-        build_config = {"read_dir": self.DATA_DIR}
 
-        solver = Solver(self.params, build_config)
+        solver = Solver(self.params, {"read_dir": self.DATA_DIR})
         solver.execute(run_mode)
 
         # Expected value comes from ./test_data/MFILE.DAT
         assert solver.params.r_tf_in_centre.value == pytest.approx(2.6354)
 
     @pytest.mark.parametrize("run_mode", [RunMode.READ, RunMode.READALL])
     def test_derived_radial_build_params_are_updated(self, run_mode):
-        build_config = {"read_dir": self.DATA_DIR}
-
-        solver = Solver(self.params, build_config)
+        solver = Solver(self.params, {"read_dir": self.DATA_DIR})
         solver.execute(run_mode)
 
         # Expected values come from derivation (I added the numbers up by hand)
         assert solver.params.r_tf_in.value == pytest.approx(1.89236)
-        assert solver.params.r_ts_ib_in.value == pytest.approx(3.47836)
-        assert solver.params.r_vv_ib_in.value == pytest.approx(4.09836)
-        assert solver.params.r_fw_ib_in.value == pytest.approx(4.89136)
-        assert solver.params.r_fw_ob_in.value == pytest.approx(12.67696)
-        assert solver.params.r_vv_ob_in.value == pytest.approx(13.69696)
+        assert solver.params.r_ts_ib_in.value == pytest.approx(3.52836)
+        assert solver.params.r_vv_ib_in.value == pytest.approx(4.14836)
+        assert solver.params.r_fw_ib_in.value == pytest.approx(4.94136)
+        assert solver.params.r_fw_ob_in.value == pytest.approx(12.72696)
+        assert solver.params.r_vv_ob_in.value == pytest.approx(13.74696)
```

Comment:
Where does the additional +0.05 come from? We've had radial build mismatches with PROCESS before, and these changes are reminiscent of those issues, if not symptomatic.

Reply:
It comes from this line: 
https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/9befa589f939fd631014a5ba69f2e1c3c894f630/bluemira/codes/process/_teardown.py#L235
which came from here
https://github.com/Fusion-Power-Plant-Framework/bluemira/pull/1489/commits/d0ad188afaa18dc124d3e3fbb71b8009a1b2cfd1

Reply:
That line looks wrong. Almost certainly. The thermal shield thickness is (inexplicably) "split" in PROCESS, in the sense that they have three or even four parameters for it (inboard, outboard, up, down). We only have one, because a thermal shield is just a thin sheet of metal with some pipes through which it is cryogenically cooled.

So out thermal shield thickness I think should just be the inboard thermal shield thickness, because `tk_ts` = `thshield_ib` = `thshield_ob`, etc. Adding them up just means doubling the thermal shield thickness in practice.

Reply:
Ok, I've just changed it to `shield_th = data["thshield_ib"]`